### PR TITLE
support GRAPH.LIST

### PR DESCRIPTION
**Pull request by AvitalFineRedis**
Add support to GRAPH.LIST command.

I compared it to rename_graph_comands because I used redis commands in the test, so it was easier that way.

### CODE REVIEW

File: redisplus/graph/commands.py [hunk 1]
```diff
@@ -61,3 +61,6 @@ def config(self, name, value=None, set=False):
             else:
                 raise DataError("``value`` can be provided only when ``set`` is True")
         return self.execute_command("GRAPH.CONFIG", *params)
+
+    def list(self):
```
Comment (chayim) [inline redisplus/graph/commands.py:65]: Gah... This is one of those things I'm torn about. list is a base type, so IMHO we shouldn't support a function with the same name as a base type. How about renaming the function **ls**. I realize this isn't ideal, but at least it's unix-like.

**Review by chayim — CHANGES_REQUESTED** [review]
Let's replace list with ls due to the reasons I mentioned. It's not ideal, but we knew this would happen.

File: redisplus/graph/commands.py (hunk 1 above)
Comment (AvitalFineRedis) [reply redisplus/graph/commands.py:159]: @chayim What do you say about `listGraphKeys`?

Comment (chayim) [reply redisplus/graph/commands.py:159]: I could accept list_keys. That's sane.

**Pull request merged**